### ISSUE TITLE
[FEATURE] Implement Stripe Payments

### ISSUE THREAD

**Issue by IAmNolanPerry** [vortexsystem/VortexWeb#8]
This is left up to the Developer, as to how they implement.

**Comment by Tampa** [comment]
It would be easier to implement an actual billing system api than try to create your own

**Comment by IAmNolanPerry** [comment]
Yes, that is what I meant, Implement the Stripe Payment System into Vortex, didn't specify the library or way to do it

**Comment by Tampa** [comment]
You would need to comply with various regulations and probably first build a single point of customer support aka email import before doing that, hence why it may make more sense to simply integrate with an already existing support and billing systems rather than building your own.

**Comment by IAmNolanPerry** [comment]
Most Stripe Integrations and Implementations are  Compliant With Regulations, because it doesn’t save any actual card details on your web server, it sends it to stripe which “tokenizes” it and you use that to reference payment sources
